types/dvtng__jss/dvtng__jss-tests.ts

```diff
@@ -0,0 +1,16 @@
+
+
+jss.set('.demo', {
```

Comment:
where does this come from? doesn't look like the right interface 😄

Reply:
Oh this comes from a different project, from Box9/jss

Reply:
Yeah, somebody was squatting on `@types/jss`, so I had to move it before I could write these.

Reply:
`foo__bar` is the style for scoped packages e.g. `@dvtng/jss` on npm. We don't really have a style for things not published to npm; I would just call this `dvtng-jss`.